include/wx/msw/webview_edge.h

```diff
@@ -0,0 +1,186 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        include/wx/msw/webview_edge.h
+// Purpose:     wxMSW Edge Chromium wxWebView backend
+// Author:      Markus Pingel
+// Created:     2019-12-15
+// Copyright:   (c) 2019 wxWidgets development team
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef wxWebViewEdgeC_H
```

Comment:
Superminor, but: the "C" was removed from everywhere else, I would recommend removing it from the header guard as well.
```suggestion
#ifndef wxWebViewEdge_H
```